File: Multi_Language_CodeExamples/scala/calculator.scala
/* Scala Functions */

object Calculator {
  def add(a: Int, b: Int): Int = a + b
  def multiply(a: Int, b: Int): Int = a * b
  
  def main(args: Array[String]): Unit = {
    println(s"5 + 3 = ${add(5, 3)}")
    println(s"5 * 3 = ${multiply(5, 3)}")
  }
}

File: Multi_Language_CodeExamples/csharp/Calculator.cs
using System;

class Calculator {
    static int Add(int a, int b) => a + b;
    static int Multiply(int a, int b) => a * b;
    
    static void Main() {
        Console.WriteLine($"5 + 3 = {Add(5, 3)}");
        Console.WriteLine($"5 * 3 = {Multiply(5, 3)}");
    }
}
